### PR TITLE
Added NodeJS Sample

### PR DESCRIPTION
**Pull request by anthony-ngu**

### CODE REVIEW

File: NodeJs.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+title: Node.js
+permalink: /NodeJS.htm
+---
+
+#Node.js Sample
+Learn to run node.js on a galileo and call APIs from a javscript file.
+
+##Running Node on Galileo
+
+* Make sure you are using the latest WIM.
```
Comment (ooeygui) [inline NodeJs.md]: Make sure you are running the latest windows image.  (Wim is something we refer to internally, but isn't referenced really anywhere).
Include a pointer to the update page.

File: NodeJs.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+* Make a directory in the root of your image named "node".
```
Comment (ooeygui) [inline NodeJs.md]: Use a sub bullet point, tells them to open \mygalileo\c$ then right click create folder called 'node'

File: NodeJs.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+* Download the 32-bit Windows Binary (.exe) of node from [nodejs.org](http://nodejs.org/download/){:target="_blank"} and place it inside the new "node" directory.
+* Download the latest zipped version of npm from [nodejs.org](http://nodejs.org/dist/npm/){:target="_blank"} and extract the contents into the new "node" directory.
+* Add these "node" folder to your Galileo image's path and restart using the following commands:
```
Comment (ooeygui) [inline NodeJs.md]: From telnet? Or from the desktop? later on you say to telnet, but you don't here...

File: NodeJs.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+~~~
+
+* Create a main.js with contents from the code section below.
+* Install the ms-iot-wiring package into the location of main.js using npm:
+
+~~~
+npm install ms-iot-wiring
+~~~
+
+* Using a telnet connection to your galileo, change directories to the location of main.js.
```
Comment (ooeygui) [inline NodeJs.md]: You are saying to run main.js but haven't filled it yet

File: NodeJs.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+* Run the command: <kbd>node main.js</kbd>
+
+##Code
+The code below shows usages of the APIs available through the nodewiring extension.
```
Comment (ooeygui) [inline NodeJs.md]: 'add the following code to main.js on the galileo'

Tip: you can open this file directly from a text editor by navigating to \mygalileo\c$\node\main.js

Comment (anthony-ngu) [inline NodeJs.md]: Is this really needed? I already said in an earlier step to Create the main.js with contents from the code section below. (Also, I think people should be able to understand that they can open up and edit the main.js after they have a file explorer window open from earlier)

File: NodeJs.md (hunk 4 above)
Comment (anthony-ngu) [inline NodeJs.md]: "Create a main.js with contents from the code section below."

Comment (ooeygui) [inline NodeJs.md]: Ok

File: NodeJs.md (hunk 5 above)
Comment (ooeygui) [inline NodeJs.md]: You never actually said how to create the file. This is the first time we've told someone to create a file and edit on the galileo - all other projects say to deploy from visual studio. I'd like to make it more explicit.

Comment (anthony-ngu) [inline NodeJs.md]: K. Added that information

**Pull request merged**